docs/source/configuration/operation-limits.mdx

```diff
@@ -123,3 +123,24 @@ You can enable or disable `warn_only` mode in your router's [YAML config file](.
 limits:
   warn_only: true # warn_only mode always enabled
 ```
+
+## HTTP Response
+
+If you have enabled operation limits to reject any operations that exceeed the configureed limits, clients will get an `HTTP 400` response with a GraphQL error.
```

Comment:
```suggestion
## Response format for exceeded limits

Whenever your router rejects a request because it exceeds an operation limit, the router responds with a 400 HTTP status code and a standard GraphQL error response body:
```